server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -100,7 +100,47 @@
           form=forms[chartId],
           csrfToken=csrfToken,
           report=incentiveLevelsTrendsByCharacteristic,
-          showPrisonPopulation=false
+          showTotalPopulation=false
+        ) }}
+
+      </section>
+    </div>
+
+    <div class="govuk-grid-row">
+      <section class="govuk-grid-column-two-thirds govuk-!-margin-bottom-9">
+
+        {% set chartId = "trends-entries-by-" + characteristicName %}
+
+        {% set afterHeaderHtml %}
+          <form id="form-select-trendsEntriesGroup" method="get" action="#heading-{{ chartId }}" novalidate>
```

Comment:
Use `action` to pass the anchor tag when submitting

Reply:
Ah yes - I misread the other bit of logic. @xoen helped me on that with his comment https://github.com/ministryofjustice/hmpps-incentives-ui/pull/192#discussion_r880586202

Updated that